### PR TITLE
Adjust style

### PR DESCRIPTION
**Pull request by DavidMatalik**
Peek 2021-05-19 19-34

What do you think about coloring?

Social icons could be improved. But it's not so easy to find and also insert them. Improve in this sprint or other sprint?

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -130,7 +130,6 @@ header h1 {
 	background: #ffffff;
 	display: inline-block;
 	margin: 0 auto 5% auto;
-	padding: 10px 10px 5px;
```
Comment (RichStone) [inline src/App.scss:133]: as mentioned, you probably want to keep some bottom padding. This is a good situation to practice rebasing. Basically, you'll need to make the change, than rebase onto this commit and fix the commit up (do a "fixup"). Then you'd make a `push --force` which would change the history. If all of it doesn't tell you anything, we can practice this later together or you can try on your own first ;)

**Review by RichStone — APPROVED** [review]
Great first fiddling with the template! 

I think we can look at the Icons later, since those are details.

I've pulled some more stuff into the sprint, feel free to adjust ;)

File: src/App.scss (hunk 1 above)
Comment (DavidMatalik) [reply src/App.scss:133]: Edit: I finally managed to fixup and rebase. =) Did it with command `git rebase --autosquash --interactive commit-hash` where commit-hash was the hash of commit right before the one I fixed before with fixup.

[I tried to follow these instructions. ](https://blog.sebastian-daschner.com/entries/git-commit-fixup-autosquash). The fixup worked, but the rebase didn't. For rebase I tried command `git rebase --autosquash --interactive base` but this threw `fatal: invalid upstream 'base'`. 

Then I tried command `git rebase --autosquash --interactive adjustStyle`. Tjhat kind of did something, but still the fixup is shown as before in commit history.

**Comment by RichStone** [comment]
Perfect, congrats to your first rebase! 😊🎉

**Pull request merged**